android/quest/src/test/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModelTest.kt

```diff
@@ -210,4 +222,77 @@ class ProfileViewModelTest : RobolectricTest() {
       profileViewModel.onEvent(event)
     }
   }
+
+  @Test
+  fun testManagingEntityShowsProfileBottomSheetFragment() = runTest {
```

Comment:
Rename test case to indicate what is actually being tested, what should be the expected behaviour?

E.g. `testThatManagingEntityProfileBottomSheetIsShownOnActionTriggered`